src/indexer/allo/v2/handleEvent.ts

```diff
@@ -352,6 +410,68 @@ export async function handleEvent(
         },
       ];
     }
+
+    case "Registered": {
+
+      const anchorAddress = parseAddress(event.params.recipientId);
+      const project = await db.getProjectByAnchor(chainId, anchorAddress)
+
+      if (!project) {
+        throw new ClientError("Project not found", 404);
+      }
+
+      const encodedData = event.params.data;
+
+      const strategyAddress = parseAddress(event.address);
+      const round  = await db.getRoundByStrategyAddress(chainId, strategyAddress);
+
+      if (!round) {
+        throw new ClientError("Round not found", 404);
```

Comment:
```suggestion
        throw new Error("Round not found");
```

client error is part of the HTTP API